lib/routes/content.js

```diff
@@ -358,10 +359,12 @@ const routes = application => {
         return next();
       }
 
-      // Validation
-      const validated = content.form.validate(req.body, type);
+      if (req.body.submit === 'Save Revision') {
```

Comment:
Yah, but we nreally really need to get these things out of config and replace them with https://github.com/punchcard-cms/punchcard/issues/63